index.html

```diff
@@ -3,9 +3,12 @@
 <head>
     <meta charset="UTF-8">
     <link rel="stylesheet" href="styles.css">
-    <title>Nep huiswerkopdracht</title>
+    <title>Nep huiswerkopdracht git workshop 2</title>
 </head>
 <body>
     <h2>Dit is een andere titel, die gaat niet over Pim</h2>
+
+<p>Dit is een onwijs gave opdracht.</p>
```

Comment:
Goed bezig, ga zo door!

Een kleine suggestie is om een argument te geven waarom je het een gave opdracht vindt: 

```suggestion
<p>Dit is een onwijs gave opdracht, omdat</p>
```